Example/Tests/NetworkTests/UploaderTests/UploaderTests.swift

```diff
@@ -1143,6 +1143,39 @@ class UploaderTests: NetworkBaseTest {
             XCTFail("Error should hold a message in its user info.")
         }
     }
+    
+    func testFilenameOverride() {
+
+        XCTAssertNotNil(cloudinary!.config.apiSecret, "Must set api secret for this test")
+
+        let expectation = self.expectation(description: "Upload should succeed and uploaded filename should change to 'overridden'")
+        let resource: TestResourceType = .borderCollie
+        let file = resource.url
+        var result: CLDUploadResult?
+        var error: NSError?
+
+        let params = CLDUploadRequestParams()
+        params.setFilenameOverride("overridden")
+        params.setUseFilename(true)
+        params.setResourceType(.image)
+        cloudinary!.createUploader().signedUpload(url: file, params: params).response({ (resultRes, errorRes) in
+            result = resultRes
+            error = errorRes
+
+            expectation.fulfill()
+        })
+
+        waitForExpectations(timeout: timeout, handler: nil)
+
+        XCTAssertNotNil(result, "result should not be nil")
+        XCTAssertNil(error, "error should be nil")
+        
+        if let filenameOverrideParam = result?.getParam(.filenameOverride) as? String {
+            XCTAssertEqual(filenameOverrideParam, "overridden", "Filename mismatch, replaced name should be 'overridden'")
+        } else {
+            XCTFail("Upload param 'filename_override' is missing")
```

Comment:
why not just assert presence of `.filenameOverride ` and then assert the value itself?

Reply:
@const-cloudinary For some reason the result param json does not contain the "filename_override" key, yet the name is correctly overriden to the desired output.
`    - key : "original_filename"
    - value : overridden`
So if the name was correctly changed the test passes, otherwise it fails